test.tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I think we can use common tsconfig for both pbiviz and tests.

Reply:
We cannot in fact because they are source of start point for webpack so, we must use diffeerent tsconfigs.